src/transit_equity/orca/query/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+This package stores common queries used on the orca_ng database
```

Comment:
It looks like this is under `orca` not `orca_ng`, so I'm confused by this comment.

Reply:
Oh that's incorrect. Thank you!

Reply:
Fixed!